<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Schedules DEMO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="date">
     <date>
      <year>2021</year>
      <month>11</month>
      <day>3</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt_browser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>421</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vesselName">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto Update</string>
    </property>
    <widget class="QComboBox" name="cmb_interval">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1분</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10분</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30분</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1시간</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10시간</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="txt_time">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_interval">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Updated:</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_status">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
